src/LineUI/index.ts

```diff
@@ -19,6 +19,7 @@ export interface IPointSet {
   showOrientation?: boolean;
   styling?: string;
   areaName?: string;
+  isUpDirection?: boolean;
```

Comment:
`isUpDirection` shouldn't be something managed within the UI kit components. I've added a longer explanation to the main review comment.

Reply:
isUpDirection  is removed from code.